src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.domain
+
+class Lotto(val numbers: Set<Int> = createLottoNumbers()) {
+    init {
+        require(numbers.size == LOTTO_NUMBER_COUNT) { "로또 당첨 번호는 6개입니다." }
+        require(checkRangeLottoNumber(numbers)) {
+            "로또 번호는 1부터 ${LOTTO_NUMBER_RANGE_END}까지 입니다."
+        }
+    }
+
+    private fun checkRangeLottoNumber(numbers: Set<Int>): Boolean {
+        return numbers
+            .filterNot { num -> num in LOTTO_NUMBER_RANGE_START until LOTTO_NUMBER_RANGE_END + 1 }
+            .isEmpty()
+    }
+
+    companion object {
+        const val LOTTO_NUMBER_RANGE_START = 1
```

Comment:
```suggestion
        return numbers
            .filterNot { num -> num in LOTTO_NUMBER_RANGE }
            .isEmpty()
    }

    companion object {
        private val LOTTO_NUMBER_RANGE = 1..45
```

상수와 Range를 적절히 사용한다면 좀 더 깔끔한 코드가 될 것 같아요 😊

Reply:
오호! 코틀린에서는 이렇게도 사용할 수 있군요!